src/app/ranking/page.tsx

```diff
@@ -0,0 +1,79 @@
+import styles from "./ranking.module.css";
+import Image from "next/image";
+import data from "../../data/currentRanking.json";
+import { Fragment } from "react";
+
+export default function RankingPage() {
+  return (
+    <section className={styles.ranking}>
```

Comment:
And then add the className to give your section the desired font : 
```suggestion
    <section className={`${styles.ranking} ${font.className}`}>
```